pages/enterprise/setup.rst

```diff
@@ -0,0 +1,115 @@
+*****
+Setup
+*****
+
+Graylog Enterprise comes as a set of Graylog server plugins which need to be
+installed in addition to the Graylog open source setup.
+
+Installation
+============
+
+Once you `purchase a license for Graylog Enterprise <https://www.graylog.org/enterprise/order>`_,
+you will get download links for the tarballs and DEB/RPM packages in the confirmation mail.
+
+Tarball
+-------
+
+The tarball includes the enterprise plugin JAR files.
+
+::
+
+  $ tar -tzf graylog-enterprise-plugins-1.0.0.tgz
+  graylog-enterprise-plugins-1.0.0/plugin/graylog-plugin-archive-1.0.0.jar
+  graylog-enterprise-plugins-1.0.0/plugin/graylog-plugin-license-1.0.0.jar
+
+Depending on the Graylog setup method you have used, you have to install the plugins into different locations.
+
+.. list-table:: Plugin Installation Locations
```

Comment:
We should also indicate that the plugins need to be installed in all Graylog servers inside the cluster. I think that is mentioned somewhere in the docs, but we should remark it here as well.